Documentation/configuring-storage.md

```diff
@@ -34,6 +34,25 @@ If you left `spec.storage.hive.s3.createBucket` set to true, or unset, then you
 Please note that this must be done before installation.
 Changing these settings after installation will result in broken and unexpected behavior.
 
+## Storing data in Azure
+
+You can also store your data in Azure blob storage, to do so, you must use an existing container.
+Edit the `spec.storage` section in the example [azure-blob-storage.yaml][azure-blob-storage-config]] configuration.
+Set the  `spec.storage.hive.azure.container` and `spec.storage.hive.azure.storageAccountName` value.
+`spec.storage.hive.azure.secretAccessKey` must also be set, however, the value can be set with an existing secret. To use an already existing secret, set createSecret to false and leave the two fields blank, with the secret following this format:
```

Comment:
set createSecret -> set `createSecret`

Reply:
We don't want users to set createSecret, so this is irrelevant